src/ripe_rainbow/domain/logic/ripe_util_vue.py

```diff
@@ -16,7 +16,7 @@ def login_wait(self, environment = "ci", redirect_url = None):
                 ("env", [environment])
             ]
         )
-        self.waits.visible(".top-bar .center .selector .title", text = environment)
+        self.waits.visible(".top-bar .center .selector  .selection", text = environment)

```

Comment:
Delete the extra space

```suggestion
        self.waits.visible(".top-bar .center .selector .selection", text = environment)
```